config/samples/capabilities_v1beta1_product_gateway_response.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: capabilities.3scale.net/v1beta1
```

Comment:
We should add this new sample in the CRD tests

Reply:
crd_validation_test.go automatically tests all samples based on file prefix name. It is already tested:

```
$ make test-crds
.....
=== RUN   TestSampleCustomResources/capabilities_v1beta1_product_gateway_response.yaml
--- PASS: TestSampleCustomResources (0.22s)
    .....
    --- PASS: TestSampleCustomResources/capabilities_v1beta1_product_gateway_response.yaml (0.00s)

```

Reply:
:+1: